src/features/factory-tycoon/components/GameCanvas.tsx

```
@@ -396,9 +425,61 @@ export function GameCanvas({
+										Empty Tile
+									</div>
+								)}
+							</div>
+							{/* Backdrop to close on click outside (scoped to canvas) */}
+							<div
+								className="fixed inset-0 z-[-1]"
+								onClick={closeContextMenu}
+								onKeyDown={(e) => {
+									if (e.key === "Escape") closeContextMenu();
+								}}
+								role="button"
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)